repo/packages/D/dcos-enterprise-cli/10/package.json

```diff
@@ -0,0 +1,9 @@
+{
+  "packagingVersion": "3.0",
+  "name": "dcos-enterprise-cli",
+  "version": "1.0.7",
```

Comment:
Do we need to bump the rev of the package ?

I think it does not matter for Linux and Mac binaries, but fixed Windows version needs to bumped hence bumping all of them..